platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -470,3 +475,14 @@ def get_thermal_manager(self):
         from .thermal_manager import ThermalManager
         return ThermalManager
 
+    def initialize_system_led(self):
+        """
+        Init system LED color to green.
+        This is for 201911 branch only
+        """
+        try:
+            with open(LED_GREEN, 'w') as f:
```

Comment:
@Junchao-Mellanox maybe it is better to have more meaningful name, e.g. `SYSTEM_STATUS_LED`. This will help to differentiate from other board leds

Reply:
Fixed